### ISSUE TITLE
G-sync turning off after a while

### ISSUE THREAD

**Issue by Woodsnaps** [elexor/nvLFCreset#2]
First off all: Great effort developing this stuff! I love it already, though, it's not working for me.

When running nvLFCreset, all is working as expected, I can confirm with the frame-times, and fps counters (both the screen built-in as well as RTSS), but after just 10 seconds, the well-known below refreshrate monitor blackout kicks in and then G-sync is turned off.

Any plans on working on this in the future? I'm glad to help if you need any.

**Comment by elexor** [comment]
Hi cheers

You may need to increase the min vrate in cru->display properties some displays will blank out below a certain hz.

lfc is pretty much required to prevent monitor blankouts when framerate dips below min hz. the program shouldn't disable lfc outright but reset it when it get's stuck on. if it's fully disabling vrr then something isn't working right.

haven't worked on this in awhile, will revisit it soon

**Comment by Woodsnaps** [comment]
> _if it's fully disabling vrr then something isn't working right._

yeah it looks like nvLFCreset crashes the G-sync drivers, might have to do with nvLFCreset not being updated lately or something.

But great work anyway, I hope you'll find time to revisit it soon, because I think this could be a huge application for future users with all the non-gsync compatible/non-freesync monitors coming up that have strobing effects because of LFC-stickiness
